bundler/lib/bundler/man/bundle-config.1.ronn

```diff
@@ -176,6 +176,9 @@ learn more about their operation in [bundle install(1)](bundle-install.1.html).
    relative paths in the `Gemfile`, among other things. By default, bundler
    will search up from the current working directory until it finds a
    `Gemfile`.
+* `gemfile_lock` (`BUNDLE_GEMFILE_LOCK`):
```

Comment:
I prefer to use `BUNDLE_LOCKFILE` instead of `BUNDLE_GEMFILE_LOCK` with same reason of https://github.com/ruby/rubygems/pull/9059/files#r2520523557